using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -309,6 +309,8 @@ If the provided URL is null or empty, it is returned as is. Otherwise, the follo
   * `TS` - A timestamp taken when this request was made
 * The optional `adobe_aa_vid` attribute is the URL-encoded Analytics Custom Visitor ID \(VID\), if previously set in the [Analytics extension](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics).
 
+When _AdobeCallbackWithError_ is provided, a 500ms timeout is used when fetching the attributes described above from the Mobile SDK. If the operation times out or an unexpected error occurs, the _fail()_ method is called with the appropriate _AdobeError_.
```

Comment:
I'm not finding a ton of examples but it looks like we commonly reference class/method names using backtick notation instead of italics. I would suggest changing _AdobeCallbackWithError_ to `AdobeCallbackWithError`. Likewise with `fail()` and `AdobeError`. Please apply throughout this page.